tests/v3/test_attributes.py

```diff
@@ -0,0 +1,13 @@
+import zarr.core
+import zarr.core.attributes
+import zarr.store
+
+
+def test_put():
```

Comment:
in principle we could test this functionality for other stores. but I also think testing the full product of `[array / group features * stores]` is excessive. I guess as long as we are confident that we are densely testing the core store APIs across store types, then it's fine to just test something derived from the store APIs on a single store.

Reply:
I think that in this case it's probably OK, since the test here is focused on the behavior of Attributes. This happens to require a reference to a Group (which requires a Store) but hopefully that interface is sufficiently covered by other tests.